ICT4DNews/app/src/main/java/at/ict4d/ict4dnews/screens/news/detail/ICT4DNewsDetailActivity.kt

```diff
@@ -58,14 +63,31 @@ class ICT4DNewsDetailActivity : BaseActivity<ICT4DNewsDetailViewModel, ActivityI
         super.onBackPressed()
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.menu_ict4_dnews_detail, menu)
+        return true
+    }
+
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         when (item.itemId) {
             android.R.id.home -> {
                 binding.fab.visible(false)
                 supportFinishAfterTransition()
                 return true
             }
+
+            R.id.action_open -> {
+                ChromeTabload()
+            }
         }
         return super.onOptionsItemSelected(item)
     }
-}
+
+    private fun ChromeTabload() {
+        val newsLink = intent.getParcelableExtra<News>(KEY_NEWS_LIST_MODEL)
+        val url = newsLink.link
+        val builder = CustomTabsIntent.Builder()
+        val customTabsIntent = builder.build()
+        customTabsIntent.launchUrl(this, Uri.parse(url))
```

Comment:
I also had to use the custom tab after I created your Issue ticket. I therefore already created a Kotlin extention. Check out the file `ICT4DExtensions.kt` and in there the function `fun Context.browseCustomTab(url: String)`. You can therefore just write `browseCustomTab(url)`. Kotlin Extentions are pretty nice 😉